frontend/src/_services/form.utils.service.ts

```diff
@@ -0,0 +1,85 @@
+import { Injectable } from '@angular/core';
+import { AbstractControl, FormGroup } from '@angular/forms';
+import { PASSWORD_LOWERCASE } from '../app/account/_validators/lowercase-password';
+import { PASSWORD_UPPERCASE } from '../app/account/_validators/uppercase-password';
+import { PASSWORD_NUMERIC } from '../app/account/_validators/numeric-password';
+import { PASSWORD_SPECIAL } from '../app/account/_validators/special-password';
+import { PASSWORD_SHORT } from '../app/account/_validators/short-password';
+import { PASSWORD_WEAK } from '../app/account/_validators/weak-password.validator';
+import { PASSWORD_MISMATCH } from '../app/account/_validators/mismatch-password.validator';
+import { USERNAME_INVALID } from '../app/account/_validators/invalid-username.validator';
+import { PASSWORD_INVALID } from '../app/account/_validators/invalid-password.validator';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class FormUtilsService {
```

Comment:
I don't really think this service is needed, but if you want to add it in, perhaps add an inline comment at the top explaining what this is for?